deploy/roles/gocd/tasks/main.yaml

```diff
@@ -10,10 +10,10 @@
     - restart go-agent
   tags: gocd
 
-- name: Copy local to remote
-  copy: 
-    src: "{{ gocd_agent_package_url }}"
-    dest: "{{ gocd_agent_package_path }}/{{ gocd_agent_package_name }}"
+- name: Download GoCD Agent RPM
+  shell: "curl -o {{ gocd_agent_package_path }}/{{ gocd_agent_package_name }} {{ gocd_agent_package_url }}"
+  args:
+    creates: "{{ gocd_agent_package_path }}/{{ gocd_agent_package_name }}"
```

Comment:
Was this a bug workaround?  Normally I try to use the Ansible module rather than `shell` whenever possible.

Reply:
Yes; this is a workaround for the "Failed to validate the SSL certificate" errors we were getting; it's simpler than trying to use Python 2.7.  It (separating download from install) pops up a couple other places, too, for the same reason.